docs/themes/global.js

```diff
@@ -5,4 +5,8 @@ export default createGlobalStyle`
     color: ${(props) => props.theme.mdxContent.textColor};
     background-color: ${(props) => props.theme.mdxContent.backgroundColor};
   }
+
+  * {
```

Comment:
`font-family` attribute shouldn't present on every DOM node. I suggest you add this rune to body.